Alert component forces me to define `description` prop (React 19)

### NextUI Version

2.6.10

### Describe the bug

When using React 18 this code is TS valid:

```tsx
<Alert>Hello</Alert>
```

After upgrading to React 19, TS compiler gives me an error, that `description` property is required:

`error TS2322: Type '{ children: string; }' is not assignable to type 'IntrinsicAttributes & MergeWithAs<Omit<DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>, "ref">, Omit<Omit<DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>, "ref">, never>, AlertProps, "div">'.
  Property 'description' is missing in type '{ children: string; }' but required in type 'Omit<UseAlertProps, "hasContent">'.`


### Your Example Website or App

_No response_

### Steps to Reproduce the Bug or Issue

See the report above.

### Expected behavior

Expected behavior in React 19 is the same like in React 18.

### Screenshots or Videos

_No response_

### Operating System Version

macOS

### Browser

Chrome

--- a/.changeset/neat-badgers-beam.md
+++ b/.changeset/neat-badgers-beam.md
@@ -1,0 +1,5 @@
+---
+"@nextui-org/alert": patch
+---
+
+mark description prop as optional (#4445)

--- a/.changeset/new-cups-tan.md
+++ b/.changeset/new-cups-tan.md
@@ -1,0 +1,5 @@
+---
+"@nextui-org/theme": patch
+---
+
+fix the alignment when only alert children is provided

--- a/packages/components/alert/src/use-alert.ts
+++ b/packages/components/alert/src/use-alert.ts
@@ -19,9 +19,9 @@ interface Props extends HTMLNextUIProps<"div"> {
    */
   title?: string;
   /**
-   * Main body of the alert message
+   * description of the alert message
    */
-  description: ReactNode;
+  description?: ReactNode;
   /**
    * Icon to be displayed in the alert - overrides the default icon
    */

--- a/packages/core/theme/src/components/alert.ts
+++ b/packages/core/theme/src/components/alert.ts
@@ -25,7 +25,8 @@ import {colorVariants} from "../utils";
 const alert = tv({
   slots: {
     base: "flex flex-grow flex-row w-full items-start py-3 px-4 gap-x-1",
-    mainWrapper: "h-full flex-grow min-h-10 ms-2 flex flex-col box-border items-start text-inherit",
+    mainWrapper:
+      "h-full flex-grow min-h-10 ms-2 flex flex-col box-border items-start text-inherit justify-center",
     title: "text-small w-full font-medium block text-inherit leading-5",
     description: "pl-[1px] text-small font-normal text-inherit",
     closeButton: "relative text-inherit translate-x-1 -translate-y-1",

